src/server/controllers/UserController.js

```diff
@@ -126,4 +126,24 @@ async function getUserfromToken(token) {
     return null;
 }
 
-module.exports = { registerUser, loginUser, tokenChecker };
\ No newline at end of file
+const getUserBio = async (req, res) => {
+    try {
+        const { userId } = req.body;
+        const usersCollection = await dbClient.getCollection('chatDB', 'users')
+        const user = await usersCollection.findOne({_id: userId});
+        const chats = await dbClient.getCollection("chatDB", "chats");
+
+        // gets all groups chats the user is member of
+        const chatGroups = chats.find({users: {$in: userId}, isRoomChat: true})
```

Comment:
what do you do you expect chatGroups to be

Reply:
It will be a list of all groups the user is part of